### DISCUSSION TITLE
How to show last page viewed after closing MAUI app with prism native way?

### DISCUSSION THREAD

**Discussion by soroshsabz** [PrismLibrary/Prism#3249] [.NET MAUI]
ITNOA

Hi

I want to find a good way to show last page viewed after re opening app, So how to show last page viewed after closing MAUI app with prism native way?

I ask same question on StackOverflow https://stackoverflow.com/q/78948191/1539100

**Comment by soroshsabz** [comment]
@dansiegel 

Do you know any clean way for achieve this?

thanks

**Reply by dansiegel** [reply]
If the app was just in the background then the state should be the same and it should just come back up on the last page. Currently the best thing to do is to use the Global Navigation Observer and maintain your own Uri. This is something I will revisit to enhance in a future release

**Reply by soroshsabz** [reply]
My app is completely closed
